### PR TITLE
Support custom colors on background selector

### PR DESCRIPTION
**Pull request by vinpac**
I will fix the failing checks later. The `cli.js` should read `tailwind.config`'s `theme.extend.colors` keys and pass it to `build.js`

### CODE REVIEW

**Comment by vinpac** [comment]
Hey, could I get a review on this? @vadimdemedes

File: cli.js [hunk 1]
```diff
@@ -17,10 +18,35 @@ const source = `
...
+			config.theme &&
+			config.theme.extend &&
+			config.theme.extend.colors
+		) {
+			return Object.keys(config.theme.extend.colors);
+		}
+	} catch (error) {
+		if (error.code !== 'MODULE_NOT_FOUND') {
+			throw error;
+		}
```
Comment (vadimdemedes) [inline cli.js:39]: Before `require()`ing a file, check for its existence first to avoid this error handling.

**Comment by vadimdemedes** [comment]
There seems to be a cleaner solution in https://github.com/vadimdemedes/tailwind-rn/pull/31. What do you think about it? Any drawbacks?

File: cli.js (hunk 1 above)
Comment (vmarcel) [reply cli.js:39]: I have the change I want to push up but I don't have permissions

**Comment by vinpac** [comment]
Hey @vadimdemedes . Could you give us a direction here? Should I make the changes or you will decide for the other PR?

File: cli.js (hunk 1 above)
Comment (vadimdemedes) [reply cli.js:39]: Feel free to fork this PR and open your own.

**Comment by vadimdemedes** [comment]
Closing in favor of https://github.com/vadimdemedes/tailwind-rn/pull/43. The linked PR should be more maintainable long-term and more flexible too. Thanks for submitting your take on this complicated issue!

**Pull request closed without merging**